### DISCUSSION TITLE
Vitepress Pagefind Plugin - Dynamic Routes paths.js

### DISCUSSION THREAD

**Discussion by Karlstens** [ATQQ/sugar-blog#180] [Q&A] [answered]
Thank you for the [Vitepress Plugin Pagefind](https://www.npmjs.com/package/vitepress-plugin-pagefind) package, it's most excellent! 

I have a question regarding [VitePress Dynamic Routes](https://vitepress.dev/guide/routing#dynamic-routes), and if they're supported? In my testing, PageFind doesn't appear to index any of the dynamically generated pages in my project, and I'm not sure as to why. 🤔

Decided to ask here, and also once again thank you for this excellent project. 🏆

**Comment by ATQQ** [comment]
"Could you provide a demo for reproduction, or your repository?"

**Reply by Karlstens** [reply, marked as answer]
Was meaning to reply - Since updating VitePress, Pagefind and vitepress-plugin-pagefind - that PageFind search is indeed working on VP Dynamic Routes.

**Reply by ATQQ** [reply]
Okay, I roughly understand the reason. In v0.4, I removed the configuration item `resultOptimization`, whose default value was `true` to automatically filter search content. It was incompatible with dynamic routes at that time.
